css/styles.css

```diff
@@ -271,6 +271,14 @@ body {
   to {opacity: 1}
 }
 
+.hidden {
+  display: none;
+}
+
+.shown {
```

Comment:
Fixed! Looks like I set the blocks to display none at line 218. I think I had been debugging something and accidentally left it in. Thankfully that's what was causing the issue with removing the hidden class and the blocks not appearing.